### DISCUSSION TITLE
Grey/Blacklist certain sites/domains for N weeks/months - whitelist others

### DISCUSSION THREAD

**Discussion by gautelund** [Cookie-AutoDelete/Cookie-AutoDelete#1645] [Q&A / Support]
I would like to set up CAD this way: I consider a lot of sites "bad". Google, Facebook, Instagram etc. etc. I would like these site's cookies to be deleted (in all Firefox Containers), say every month. (I don't want to be cookie-nagged by e.g. YouTube and google after every restart. I also use Google Container and Facebook Container addons, isolating Facebook and Google from other web activity, and I use Ghostery and AdBlock Plus. So I will tolerate the badlisted sites to have their cookies for a while.

All other sites should be whilelisted. 

The app is daunting in it's options. I haven't been able to make this happen.

**Comment by kennethtran93** [comment]
we don't have any easy way to 'blacklist' or 'blocklist' something yet, nor do we have a way to determine when the cookie was created without keeping track of said cookie ourselves.
